demisto_sdk/__main__.py

```diff
@@ -3104,6 +3113,126 @@ def update_content_graph(
         )
 
 
+@main.command()
+@click.help_option("-h", "--help")
+@click.option(
+    "-i",
+    "--input",
+    help="The path to the input file to run the command on.",
+    multiple=True,
+    type=click.Path(path_type=Path),
+)
+@click.option(
+    "-g",
+    "--use-git",
+    help="Whether to use git to determine which files to run the command on",
+    is_flag=True,
+    default=False,
+)
+@click.option(
+    "-s",
+    "--staged",
+    help="Whether to run on staged files only",
+    is_flag=True,
+    default=False,
+)
+@click.option(
+    "-a",
+    "--all-files",
+    help="Whether to run on all files",
```

Comment:
all files where? in the repo? in curernt folder?

Reply:
All files of the repo (similar to -a)